streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.java

```diff
@@ -1230,17 +1219,6 @@ public boolean paranoidFileChecks() {
         return columnFamilyOptions.paranoidFileChecks();
     }
 
-    @Override
-    public Options setMaxWriteBufferNumberToMaintain(final int maxWriteBufferNumberToMaintain) {
-        columnFamilyOptions.setMaxWriteBufferNumberToMaintain(maxWriteBufferNumberToMaintain);
-        return this;
-    }
-
-    @Override
-    public int maxWriteBufferNumberToMaintain() {
-        return columnFamilyOptions.maxWriteBufferNumberToMaintain();
-    }
-
```

Comment:
Same here. 
But here it is even a bit more complicated. It seems that the option that replaces this option (`max_write_buffer_size_to_maintain`) does not exist in the Java API. That means, we cannot state in the warn message to use the other option instead. Could you please research why `max_write_buffer_size_to_maintain` does not exist in the Java API? RocksDB's mailing list is a good place to start the research.